modules/extensions-spring-support/src/test/java/org/mule/module/extension/internal/capability/xml/XmlModelEnricherTestCase.java

```diff
@@ -41,11 +43,12 @@
     @Test
     public void enrich()
     {
+        declarationDescriptor.named(EXTENSION_NAME).onVersion(EXTENSION_VERSION);
         XmlModelProperty xmlProperty = enrich(XmlSupport.class);
 
         assertThat(xmlProperty, is(notNullValue()));
         assertThat(xmlProperty.getSchemaVersion(), is(SCHEMA_VERSION));
```

Comment:
This is just information to fill the declarationDescriptor in order to test, if no Name/Version are specified both will be null values that is a case that is never going to happen when building an extension.